plugins/modules/ssh_config.py

```diff
@@ -274,6 +282,9 @@ def ensure_state(self):
             controlpersist=fix_bool_str(self.params.get('controlpersist')),
             dynamicforward=self.params.get('dynamicforward'),
         )
+        if self.params.get('other_options'):
+            for key, value in self.params.get('other_options').items():
```

Comment:
Generally this should reject adding keys to `args` that are already there.

Reply:
Have added a check to see if values are already present before adding them